### PR TITLE
Add Nested Template References

### PR DESCRIPTION
**Pull request by DoubleA-MG**
Changed a small mention in the Modeling Reference for nested templates. The Template Reference seems pretty lengthy already so wasn't sure if the position or length of my H2 on Nested Templates was appropriate.

### CODE REVIEW

File: src/references/templates.md [hunk 1]
```diff
@@ -186,6 +186,32 @@ To learn more about importing and using **Community Content** templates, see the
...
+
+One example of using nested templates is for a racing game that has different car templates. If the creator wanted to add a dirt decal to the tires of all the car templates, then it would require updating each car template one by one. Using nested templates can shorten this to just one action by using a tire template nested within the car template. Updating the tire template would change all instances of the tire even across the different parent car templates.
+
+<div class="mt-video" style="width:100%">
+    <video autoplay muted playsinline controls loop class="center" style="width:100%">
+        <source src="/img/EditorManual/Art/NestedTemplates.mp4" type="video/mp4" />
+    </video>
+</div>
+
+!!! warning "Templates can only be nested up to 16 levels deep."
```
Comment (CommanderFoo) [inline src/references/templates.md]: I would add the warning above the video.

File: src/references/templates.md [hunk 2]
```diff
@@ -186,6 +186,32 @@ To learn more about importing and using **Community Content** templates, see the
...
+
+A nested template can be created by dragging one template into another in the **Hierarchy** window. The parent template needs to be deinstanced in order for the child template to be added.
+
+### Updating Nested Templates
+
+When updating a child template inside another template, there is an option of which template to update. This can be useful to change the child template in all instances or to change an individual child template property (e.g. only the front right tires are yellow for all taxi cars).
+
+![!Nested Template](../img/EditorManual/Art/NestedTemplate.png){: .center loading="lazy" }
+
+!!! info "If the child template is deinstanced when the parent template is updated, then the child template is abandoned."
```
Comment (CommanderFoo) [inline src/references/templates.md:211]: Info above the image so it's more visible. Not sure on what our style rules are for this. Might be something we need to discuss with Zaz.

File: src/references/templates.md [hunk 3]
```diff
@@ -186,6 +186,32 @@ To learn more about importing and using **Community Content** templates, see the
...
+
+!!! warning "Templates can only be nested up to 16 levels deep."
+
+### Creating Nested Templates
+
+A nested template can be created by dragging one template into another in the **Hierarchy** window. The parent template needs to be deinstanced in order for the child template to be added.
+
+### Updating Nested Templates
+
+When updating a child template inside another template, there is an option of which template to update. This can be useful to change the child template in all instances or to change an individual child template property (e.g. only the front right tires are yellow for all taxi cars).
```
Comment (CommanderFoo) [inline src/references/templates.md]: e.g. -> for example

We aren't using abbreviations in cases such as this anymore.

**Review by CommanderFoo — CHANGES_REQUESTED** [review]
Reviewed just what was added.

File: src/references/templates.md (hunk 2 above)
Comment (DoubleA-MG) [reply src/references/templates.md:211]: I think you are right, I'll keep that in mind moving forward.

**Review by CommanderFoo — APPROVED** [review]
